test/users.js

```diff
@@ -294,3 +313,162 @@ describe('/users/sessions POST', () => {
       });
   });
 });
+
+describe('/users GET', () => {
+  it('should pass getting users, token is provided', () => {
+    const page = 1;
+
+    // The users are created manually using the model to not depend on the
+    // Sign Up endpoint implementation
+    return User.create(someUser)
+      .then(user => {
+        return User.create(someUser2);
+      })
+      .then(user => {
+        // The sign in is manually to no depend on the Sign In endpoint
+        // implementation
+        return new Promise((resolve, reject) => {
```

Comment:
rather than doing this, not only here but in every place you use this logic, create a function somewhere with this and just import and call it here and wherever you need it